docs/stale-subvolume.md

```diff
@@ -0,0 +1,32 @@
+# stale-subvolume cleanup
+
+The subvolume command is used to clean the stale subvolumes
+which has no parent-pvc attached to it.
+The command would list out all such subvolumes which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the stale subvolumes that can be removed
+* `clean` : [clean](#clean) stale subvolumes after user's confirmation
+
+## ls
+
+```bash
+kubectl rook-ceph stale-subvolume ls
+
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110004
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110005
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110006
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110007
+
+```
+
+## clean
+
+```bash
+kubectl rook-ceph stale-subvolume clean csi-vol-427774b4-340b-11ed-8d66-0242ac110004
```

Comment:
What about this command structure? This gives the flexibility of being able to include all subvolumes, not just the stale subvolumes if desired. We can just document that it will only handle stale volumes by default. IMO The "stale" doesn't need to be part of the command name.
```
kubectl rook-ceph subvolume ls [--all]
kubectl rook-ceph subvolume delete <filesystem> <subvolumeName>
```

The `--all` flag seems useful for listing all the subvolumes, instead of only stale subvolumes.

Reply:
or 
`kubectl rook-ceph subvolume ls [--stale]`\ `kubectl rook-ceph subvolume ls [-s]` to just list the stale subvolumes.  

`kubectl rook-ceph subvolume ls` just returning the stale subvolumes might be a bit confusing.